ceph-container-build-ceph-base-push-imgs/config/definitions/ceph-container-build-push-imgs.yml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: ceph-container-build-ceph-base-push-imgs
+    node: huge && trusty && x86_64
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph-container: build and push ceph base container images to Docker Hub'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: -1
```

Comment:
`-1` means this will be kept forever. Please don't keep logs and build artifacts forever, set a reasonable number in days or builds that you need and set them here

Reply:
Right, thanks, let me adjust that.